app/controllers/agenda/minutes.js

```diff
@@ -36,29 +37,81 @@ function renderAttendees(attendees) {
   `;
 }
 
-function renderNotas(notas) {
-  return renderAgendaitemList(notas);
+async function renderNotas(notas, betreftPieceParts, intl) {
+  const agendalist = await renderAgendaitemList(notas, betreftPieceParts, intl);
+  return agendalist;
 }
 
-function renderAnnouncements(announcements) {
+async function renderAnnouncements(announcements, betreftPieceParts, intl) {
   return `
     <h4><u>MEDEDELINGEN</u></h4>
-    ${renderAgendaitemList(announcements)}
+    ${await renderAgendaitemList(announcements, betreftPieceParts, intl)}
   `;
 }
 
-function renderAgendaitemList(agendaitems) {
-  return agendaitems
-    .map(
-      (agendaitem) => `
-        <h4><u>${
-          agendaitem.number
-        }. ${agendaitem.shortTitle.toUpperCase()}</u></h4>
-        ${agendaitem.title ? `<p>${agendaitem.title}</p>` : ''}
-      `
-    )
-    .join('');
+async function renderAgendaitemList(agendaitems, betreftPieceParts, intl) {
+  let agendaitemList = "";
+  for (const agendaitem of agendaitems) {
+    agendaitemList += await getMinutesListItem(betreftPieceParts, agendaitem, intl);
+  }
+  return agendaitemList;
+}
+async function getMinutesListItem(betreftPieceParts, agendaitem, intl) {
+  const betreftPiecePart = betreftPieceParts.find(piecePart => piecePart.agendaitemID === agendaitem.id);
+  const treatment = await agendaitem.treatment;
+  const decisionActivity = await treatment?.decisionActivity;
+  const decisionResultCode = await decisionActivity?.decisionResultCode;
+  let mededelingOrNota = "";
+  if (agendaitem.type?.get('uri') === constants.AGENDA_ITEM_TYPES.ANNOUNCEMENT) {
+    mededelingOrNota = "deze mededeling";
+  } else {
+    mededelingOrNota = "dit punt";
+  }
+  let text = "";
+  switch (decisionResultCode?.uri) {
+    case constants.DECISION_RESULT_CODE_URIS.GOEDGEKEURD:
+      if (betreftPiecePart) {
+        text = intl.t("minutes-approval", {
+          mededelingOrNota: capitalizeFirstLetter(mededelingOrNota),
+          reportName: betreftPiecePart['reportName']
+        })
+      }
+      break;
+    case constants.DECISION_RESULT_CODE_URIS.INGETROKKEN:
+      text = intl.t("minutes-retracted", {
+        mededelingOrNota: capitalizeFirstLetter(mededelingOrNota)
+      })
+      break;
+    case constants.DECISION_RESULT_CODE_URIS.KENNISNAME:
+      text = intl.t("minutes-acknowledged", {
+        mededelingOrNota: capitalizeFirstLetter(mededelingOrNota)
```

Comment:
**chore**: don't call `capitalizeFirstLetter` here

The string ends up looking like this: `De Vlaamse Regering neemt akte van Deze mededeling.`